### PR TITLE
[Admin][Customer] Browsing orders of a customer

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/config/routing/customer.yml [hunk 1]
```diff
@@ -35,3 +35,20 @@ sylius_admin_request_customer_password_reset:
...
+            section: admin
+            template: "@SyliusAdmin/Crud/index.html.twig"
+            grid: sylius_admin_customer_order
+            vars:
+                route:
+                    parameters:
+                        $customerId: $id
+                templates:
+                    breadcrumb: '@SyliusAdmin/Customer/Order/Index/_breadcrumb.html.twig'
+            permission: true
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/AdminBundle/Resources/config/routing/customer.yml]: Not needed anymore.

File: src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_header.html.twig [hunk 2]
```diff
@@ -14,12 +14,20 @@
         <div class="right aligned column" id="actions">
             {{ render(path('sylius_admin_request_customer_password_reset', { 'id': customer.id, 'email': customer.email })) }}
             {{ buttons.delete(path('sylius_admin_shop_user_delete', {'id': customer.user.id })) }}
+
+            {% if customer.orders is not empty %}
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_header.html.twig]: This if will fetch all orders, I'd remove it and simply add the button anyway.

Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_header.html.twig]: Does it even work? See http://twig.sensiolabs.org/doc/tests/empty.html

Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_header.html.twig]: `{% if customer.orders|length > 0 %}` will work and won't fetch all orders.

**Comment by pjedrzejewski** [comment]
Thank you Grzesiu!

**Pull request merged**